### DISCUSSION TITLE
你们的PCL2主页是什么啊

### DISCUSSION THREAD

**Discussion by gyc123456-1** [Meloong-Git/PCL#2100] [讨论]
![image](https://github.com/Hex-Dragon/PCL2/assets/69791212/fdc0830f-3e22-4b26-8bd4-beebd1fe1864)
我的垃圾主页，你们都用的什么啊，说出来让我羡慕羡慕

**Comment by WForst-Breeze** [comment]
新闻主页（
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/f539d1ce-b447-47f3-a538-23966a121562)

**Reply by PlayMcBKuwu** [reply]
对，Microsoft 后遗症（

**Reply by Light-Beacon** [reply]
> 我觉得新闻主页唯一的问题就是 英文 / 数字和汉字中间没有空格，看着难受死了……（）

草我才看见 回头我看看能优化下吗XD

**Comment by MoYuan-CN** [comment]
服务器的（不是打广告哈
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/cae4149c-1d87-4f50-bddc-2465cf33f95a)

**Comment by naiximcn** [comment]
? 主页 主业是什么 PCL是什么 能吃吗 一片空白不好吗 [doge]

**Comment by yznqzty** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/135207045/2dea3629-1e91-49a8-8a50-17f8935b6a3c)

**Comment by SuoHouDaiShi** [comment]
给我服务器整的（也**不是广告**
![QQ截图20230815162145](https://github.com/Hex-Dragon/PCL2/assets/119282947/106baee9-217f-41ef-b65e-b7063384297e)

**Reply by PlayMcBKuwu** [reply]
如何砍掉设置页面（雾

**Reply by SuoHouDaiShi** [reply]
> 如何砍掉设置页面（雾

功能隐藏啦

**Comment by Chiloven945** [comment]
🤔
![image](https://github.com/Hex-Dragon/PCL2/assets/116699482/897d8250-ab7e-47a7-a13c-0cf8facb9745)
